README.md

```diff
@@ -1,4 +1,40 @@
-# GitOps Flux Configuration
+# Flux Configuration
 
-GitOps Flux Configuration is reusable Configuration designed to be primarily used in
-higher level Configurations.
+
+This repository contains a [Crossplane configuration](https://docs.crossplane.io/v1.11/concepts/packages/#configuration-packages), tailored for users establishing their initial control plane with [Upbound](https://cloud.upbound.io). This configuration deploys fully managed Flux instances allowing you to integrate GitOps practices into your workflow.
```

Comment:
can we use here https://docs.crossplane.io/latest/concepts/packages/#configuration-packages instead of `v1.11`